### PR TITLE
COMMERCE-9601

### PR DESCRIPTION
**Pull request by Brunno0liveira**
[COMMERCE-9601](https://liferay.atlassian.net/browse/COMMERCE-9601)

### CODE REVIEW

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommercePaymentMethodsPermissions.testcase [hunk 1]
```diff
@@ -195,6 +195,140 @@ definition {
...
+
+			Role.viewPermissionPG(
+				permissionDefinitionKey = "INLINE_PERMISSIONS_PERMISSIONS_CHECKBOX",
+				permissionFlagged = "true",
+				roleTitle = "Operations Manager");
+
+			IFrame.closeFrame();
+		}
+
+		task ("And on Money Order payment method, View permission is not set for Operations Manager") {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommercePaymentMethodsPermissions.testcase:257]:
```suggestion
		task ("And on Money Order payment method, Permissions permission is not set for Operations Manager") {
```

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
Issues are not related to the changes, PR can be forward

@victoroscuro

**Comment by victoroscuro** [comment]
ci:forward:force

**Pull request closed without merging**